### PR TITLE
Data Join can occur off device

### PR DESCRIPTION
**Pull request by benjaminsavage**
The group just discussed this aspect of the design dimension and there was no objection (philosophically) to the idea of source and trigger events being joined (attribution being performed) server side.

### CODE REVIEW

File: design-dimensions/Dimensions-with-General-Agreement.md [hunk 1]
```diff
@@ -41,3 +41,7 @@ In order to be useful, measurement systems need to produce a continuous release
 The community group has reached general agreement that the _Private Measurement Technical Specification MVP_ should _at least_ include a time or interaction component to the budget scope. That is, as the user browses the web and interacts with sites more, we would expect that the private measurement system will allow more information about that user to be disclosed.
 
 It is worth noting that, at least among existing differential privacy deployments, a time dimension in the privacy unit is very common. See https://desfontain.es/privacy/real-world-differential-privacy.html for a list of examples here.
+
+## Location of Data Join
+
+The community group has reached general agreement that data join could potentially occur off device within a some type of server side architecture that achieves our desired security goals.
```
Comment (martinthomson) [inline design-dimensions/Dimensions-with-General-Agreement.md:47]:
```suggestion
The community group has reached general agreement that data join could potentially occur off device within a some type of server-side architecture that achieves our desired security goals.
```

...I couldn't resist.

**Review by martinthomson — APPROVED** [review]

File: design-dimensions/Dimensions-with-General-Agreement.md (hunk 1 above)
Comment (csharrison) [reply design-dimensions/Dimensions-with-General-Agreement.md:47]:
```suggestion
The community group has reached general agreement that data join could potentially occur off device within some type of server-side architecture that achieves our desired security goals.
```
One more nit on this line.

Comment (martinthomson) [reply design-dimensions/Dimensions-with-General-Agreement.md:47]: I might phrase this slightly differently though.

```suggestion
The community group has reached general agreement that data join could potentially occur off device within a some type of server-side architecture.
This is conditional on having adequate protections for any data that leaves a device, in line with our security and privacy goals.
```

**Comment by csharrison** [comment]
High level, did you consider moving this as a subsection under the "Server Architecture may exist" header? It seems somewhat natural for that to describe a section of what server architectures are acceptable.

**Comment by benjaminsavage** [comment]
Makes sense - happy for this to be under that section

**Comment by benjaminsavage** [comment]
I don't know how to edit commits made through the Github UI, so I just made a new one with ya'lls proposed changes =).

https://github.com/patcg/docs-and-reports/pull/48

**Pull request closed without merging**